cap_anndata_schema.md

```diff
@@ -0,0 +1,1577 @@
+* <code>[EMBEDDING TYPE]</code>: MUST denote the algorithm used to generate the embedding (e.g. `UMAP`, `tSNE`, `pca`, etc.). REQUIRED.
+* <code>[SUFFIX]</code>: Denotes a descriptive tag informative enough for third-party users; used to distinguish between multiple embeddings of the same type. OPTIONAL.
+
+**examples:** <td><code>'X_pca'</code>, <code>'X_tsne'</code>, <code>'X_tSNE'</code>, <code>'X_umap'</code>, <code>'X_UMAP_nneigbors15'</code>, <code>'X_umap_2'</code>
+
+
+# uns (Dataset metadata)
+
+**NOTE:** Each time a cell annotation `cellannotation_setname` is modified, these values potentially change. 
+
+## cellannotation_schema_version
+
+Key-value pair in the `uns` dictionary
+
+<table><tbody>
+	<tr>
+  		<td><b>key</b></td>
+  		<td><code>cellannotation_schema_version</code></td>
+	</tr>
+	<tr>
+  		<td><b>type</b></td>
+  		<td>string</td>
+	</tr>
+	<tr>
+  		<td><b>value</b></td>
+  		<td>The schema version, the cell annotation open standard. 
+This versioning MUST follow the format <code>'[MAJOR].[MINOR].[PATCH]'</code> as defined by <a href="https://semver.org">Semantic Versioning 2.0.0.</a> Current version MUST follow 0.1.0</td>
+	</tr>
+	<tr>
+  		<td><b>source</b></td>
+  		<td>software</td>
+	</tr>
+	<tr>
+  		<td><b>required for publication on CAP</b></td>
+  		<td>yes</td>
+	</tr>
+	<tr>
+  		<td><b>example</b></td>
+  		<td><code>'0.1.0'</code></td>
+	</tr>
+</tbody></table>
+
+
+## cap_dataset_timestamp
+
+Key-value pair in the `uns` dictionary
+
+<table><tbody>
+	<tr>
+  		<td><b>key</b></td>
+  		<td><code>cap_dataset_timestamp</code></td>
+	</tr>
+	<tr>
+  		<td><b>type</b></td>
+  		<td>string</td>
+	</tr>
+	<tr>
+  		<td><b>value</b></td>
+  		<td>The timestamp of the dataset published on CAP. This MUST be a string in the format <code>%yyyy-%MM-%dd'T'%hh:%mm:%ss</code>.</td>
+	</tr>
+	<tr>
+  		<td><b>source</b></td>
+  		<td>software</td>
+	</tr>
+	<tr>
+  		<td><b>required for publication on CAP</b></td>
+  		<td>yes</td>
+	</tr>
+	<tr>
+  		<td><b>example</b></td>
+  		<td><code>'2023-11-21T04:12:36'</code></td>
+	</tr>
+</tbody></table>
+
+## cap_dataset_version
+
+Key-value pair in the `uns` dictionary
+
+<table><tbody>
+	<tr>
+  		<td><b>key</b></td>
+  		<td><code>cap_dataset_version</code></td>
+	</tr>
+	<tr>
+  		<td><b>type</b></td>
+  		<td>string</td>
+	</tr>
+	<tr>
+  		<td><b>value</b></td>
+  		<td>The version for all cell annotations published (per dataset) on CAP. This MUST be a string. The recommended versioning format is <code>'[MAJOR].[MINOR].[PATCH]'</code> as defined by <a href="https://semver.org">Semantic Versioning 2.0.0.</a></td>
+	</tr>
+  		<td><b>source</b></td>
+  		<td>software</td>
+	</tr>
+	<tr>
+  		<td><b>required for publication on CAP</b></td>
+  		<td>yes</td>
+	</tr>
+	<tr>
+  		<td><b>example</b></td>
+  		<td><code>'0.1.0'</code></td>
+	</tr>
+</tbody></table>
+
+## cap_dataset_title
```

Comment:
In the previous version it was just `dataset_title`. I prefer more simple `title` to allign with CxG. What is the best name here? CC @evanbiederstedt @mfutey

Reply:
I updated the field names to match those of the private repo, but we should update both versions to have more consistent names, e.g. `'dataset_title'`, `'dataset_description'` and `'dataset_url'` do not have the `cap_` prefix as the other fields at the dataset level.

Reply:
Agreed, this is something @dosumis has been pushing as well.